dataflux_core/performance_tests/list_and_download.py

```diff
@@ -118,6 +119,7 @@ def run_download(self, config, list_result):
             raise AssertionError(
```

Comment:
The check on the total download time and timeout doesn't have the "segmented" logic so the "download_timeout" is implicitly applied on a chunk of the download. Should we apply the similar logic here and check for the total download time in `test_list_and_download_one_shot`? Was that planned in a different PR?

Reply:
Yes, i'm going to ditch the timeout and gather metrics in the future so i didn't bother addressing.

Reply:
I'll update to a singular timeout in that cl